runagent/sdk/server/framework/__init__.py

```
@@ -15,20 +16,20 @@
 
 
 def get_executor(
-    agent_dir: Path, framework: str, agent_entrypoints: t.Dict[str, t.Union[PythonicEntryPoint, WebHookEntryPoint]]
+    agent_dir: Path, framework: Framework, agent_entrypoints: t.Dict[str, t.Union[PythonicEntryPoint, WebHookEntryPoint]]
```

Ruff (0.13.1)

Unused function argument: `agent_entrypoints`

(ARG001)